telex/client.py

```diff
@@ -48,6 +48,6 @@ def _post(self, path, payload):
             response = self.session.post('{}{}'.format(self.telex_url, path), data=json.dumps(payload))
             response.raise_for_status()
         except requests.exceptions.RequestException as e:
-            raise RequestException(request=e.request, response=e.response)
+            raise RequestException(response=getattr(e, 'response', None))
```

Comment:
i don't know that its ever stored the constructed request object in its exceptions. maybe would be useful for debugging though.